script.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+    alert('Hello,world!');
+});
+
+
+$('#test').html;
+
+$(document).ready(function() {
```

Comment:
we should only have one document.ready() function per script. That being said - you are setting the style of your mcc class! Awesome! Now - how would you use this function to also set the text of the name spans and the hobby list?